requirements.txt

```diff
@@ -25,6 +25,7 @@ hf_transfer
 sentencepiece
 gradio==3.50.2
 
+modal==0.70.5
```

Comment:
FYI we're currently on a six-month deprecation cycle for our client SDK, so this version will no longer be supported around the middle of 2025.

That's not so bad here, since users will be running one-off jobs rather than deploying apps. And I also imagine you have folks mostly running the latest versions of axolotl.